bls_thresholdsign.go

```diff
@@ -339,20 +347,20 @@ func (s *blsThresholdSignatureInspector) TrustedAdd(orig int, share Signature) (
 }
 
 // VerifyAndAdd verifies a signature share (same as `VerifyShare`),
-// and may or may not add the share to the local pool of shares.
-// This function is thread safe and locks the internal state.
+// and attempts to add the share to the local pool of shares.
+// This function is thread-safe and locks the internal state.
 //
-// The share is only added if the signature is valid, the signer index is valid and has not been
-// added yet. Moreover, the share is added only if not enough shares were collected.
-// Boolean returns:
-//   - First boolean output is true if the share is valid and no error is returned, and false otherwise.
-//   - Second boolean output is true if enough shares were collected and no error is returned, and false otherwise.
+// The share is only added if the signature is valid, the signer index is valid,
+// and has not been added yet. Moreover, the share is added only if not enough shares were collected.
```

Comment:
```suggestion
// and it has not been added yet. Moreover, the share is NOT added if enough shares for reconstruction were already collected.
```
I think this is equivalent to what you have and reads more clearly to me.